1.7/1.7.5/index.html

```diff
@@ -0,0 +1,93 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<title>Title</title>
+	<style>
+		table {
+			border-collapse: collapse;
+		}
+
+		td,
+		th {
+			border: 1px solid black;
+			padding: 3px;
+			text-align: center;
+		}
+
+		th {
+			font-weight: bold;
+			background-color: #E6E6E6;
+		}
+	</style>
+</head>
+<body>
+### 1.7.5
+Напишите функцию createCalendar(elem, year, month).
+
+Вызов функции должен создать календарь для заданного месяца month в году year и вставить его в elem.
+
+Календарь должен быть таблицей, где неделя – это tr, а день – это td. У таблицы должен быть заголовок с названиями
+дней недели, каждый день – th, первым днём недели должен быть понедельник.
+
+Например, createCalendar(cal, 2012, 9) сгенерирует в cal следующий календарь:
+
+P.S. В этой задаче достаточно сгенерировать календарь, кликабельным его делать не нужно.
+
+<div id="calendar"></div>
+
+<script>
+	const calendar = document.getElementById('calendar');
+
+	function createCalendar(year, month) {
```

Comment:
the function should take three arguments in the requirements:
![image](https://github.com/tacya9/JS-2-1/assets/115877101/17dd1f38-656d-4a76-9139-10e7bfcbbbab)